website/docs/topics/feature-flags/feature-flag-best-practices.md

```diff
@@ -1,339 +1,188 @@
 
-Feature flags have become a central part of the DevOps toolbox along with Git, CI/CD and microservices. You can write modern software without all of these things, but it sure is a lot harder, and a lot less fun.
+With feature flags, developers can turn specific features or code segments on or off at runtime without needing a code deployment or rollback. Organizations that adopt feature flags see improvements in key DevOps metrics like lead time, deployment frequency, change failure rate, and recovery time.
 
-And just like the wrong Git repo design can cause interminable headaches, getting the details wrong when first building a feature flag system can be very costly.
+At Unleash, we've defined 11 principles for building a large-scale feature flag system. These principles have their roots in distributed systems design and focus on security, privacy, and scalability—critical needs for enterprise systems. By following these principles, you can create a feature flag system that’s reliable, easy to maintain, and capable of handling heavy loads.
 
-This set of principles for building a large-scale feature management platform is the result of thousands of hours of work building and scaling Unleash, an open-source feature management solution used by thousands of organizations.
-
-Before Unleash was a community and a company, it was an internal project, started by [one dev](https://github.com/ivarconr), for one company. As the community behind Unleash grew, patterns and anti-patterns of large-scale feature flag systems emerged. Our community quickly discovered that these are important principles for anyone who wanted to avoid spending weekends debugging the production system that is supposed to make debugging in production easier.
-
-“Large scale” means the ability to support millions of flags served to end-users with minimal latency or impact on application uptime or performance. That is the type of system most large enterprises are building today and the type of feature flag system that this guide focuses on.
-
-Our motivation for writing these principles is to share what we’ve learned building a large-scale feature flag solution with other architects and engineers solving similar challenges. Unleash is open-source, and so are these principles. Have something to contribute? [Open a PR](https://github.com/Unleash/unleash/pulls) or [discussion](https://github.com/orgs/Unleash/discussions) on our Github.
-
-## 1. Enable run-time control. Control flags dynamically, not using config files.
+These principles are:
 
-A scalable feature management system evaluates flags at runtime. Flags are dynamic, not static. If you need to restart your application to turn on a flag, you are using configuration, not feature flags.
+1. [Enable runtime control: Control flags dynamically instead of through config files.](#1-enable-runtime-control)
```

Comment:
@riboflavin in your PR you used absolute links, like [1\. Enable runtime control. Control flags dynamically, not using config files.](https://docs.getunleash.io/topics/feature-flags/feature-flag-best-practices\#1-enable-run-time-control-control-flags-dynamically-not-using-config-files) - was there an SEO reason behind, or can we just go for a relative link, as the one I used in this line

Reply:
Relative is good, @melindafekete 👍🏼